test/check-cli

```diff
@@ -28,6 +28,9 @@ class TestImages(composertest.ComposerTestCase):
     def test_tar(self):
         self.runCliTest("/tests/cli/test_compose_tar.sh")
 
+    def test_tar_kickstart(self):
+        self.runCliTest("/tests/cli/test_compose_tar_kickstart.sh")
+
```

Comment:
Currently the fedora-30 scenario times out so I think tar tests should be separated into their own scenario.

Reply:
Done.